src/number_theory/bernoulli.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2020 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+
+import data.rat
+import data.fintype
+
+/-!
+# Bernoulli numbers
+
+The Bernoulli numbers are a sequence of numbers that frequently show up in number theory.
+
+For example, they show up in the Taylor series of many trigonometric and hyperbolic functions,
+and also as (integral multiples) of special values of the Riemann zeta function.
```

Comment:
```suggestion
and also as (integral multiples of) special values of the Riemann zeta function.
```

Reply:
In fact, it's not integral multiples, there is a power of \pi...

Reply:
Ooh, right, silly me.

Reply:
Fixed